src/edu/csus/ecs/pc2/core/PacketHandler.java

```diff
@@ -3602,11 +3602,9 @@ private boolean isThisSite(Account account) {
     }
 
     /**
-     * Loads only the settings from the remote server into this server.
+     * Loads only the settings from the remote server into this server's model.
      * 
-     * Loads settings from other server, submissions, etc.  Sends
-     * out a login packet to any "new" servers which this site
-     * is not logged into. 
+     * Loads settings from other server, submissions, etc. into model.
```

Comment:
This is a trivial and certainly non-blocking note, but:  I don't understand why both lines 3605 and 3607 are present; they seem to say the same thing.